tools/0a-multi-lpar-preseed/templates/preseed.cfg.jinja

```diff
@@ -0,0 +1,134 @@
+              .                                               \
+d-i partman-auto/choose_recipe select basic-boot
+d-i partman-partitioning/confirm_write_new_label boolean true
+d-i partman/choose_partition select finish
+d-i partman/confirm boolean true
+d-i partman/confirm_nooverwrite boolean true
+
+# network console password
+d-i network-console/password password pass4inst
+d-i network-console/password-again password pass4inst
+d-i network-console/start note
+
+# default user and password
+d-i passwd/user-fullname string ubuntu
+d-i passwd/username string ubuntu
+d-i passwd/user-password password {{ UBUNTU_PASSWD }}
+d-i passwd/user-password-again password {{ UBUNTU_PASSWD }}
+d-i user-setup/encrypt-home boolean false
+
+# timezone
+d-i clock-setup/utc boolean true
+d-i mirror/country string manual
+
+# installation source location
+d-i mirror/protocol string http
+d-i mirror/http/hostname string us.ports.ubuntu.com
+d-i mirror/http/directory string /
+d-i mirror/http/proxy string
+d-i mirror/suite string xenial
+
+# install stuff
+d-i live-installer/enable boolean false
+d-i tasksel/first multiselect standard system utilities, OpenSSH server, Basic Ubuntu server
+
+# late_commands
+# this is where the magic happens....
+d-i preseed/late_command string \
+# activate 2nd interface and set to layer 2 bridge
+in-target chzdev --no-root-update -pVe {{ OSA2_DEVNO }} bridge_role=primary; \
+in-target chzdev -p {{ OSA2_DEVNO }} layer2=1; \
+# configure 2nd interface for multiple vlans by stuffing the config manually
+# Note: the following is for POK Solution Test -- custom private networking
+#touch /target/etc/network/interfaces.d/private_int; \
+#echo -e "auto {{ OSA2_ENCNO }}" > /target/etc/network/interfaces.d/private_int; \
+#echo -e "iface {{ OSA2_ENCNO }} inet manual" >> /target/etc/network/interfaces.d/private_int; \
+#echo -e "" >> /target/etc/network/interfaces.d/private_int; \
+#echo -e "auto {{ OSA2_ENCNO }}.1295" >> /target/etc/network/interfaces.d/private_int; \
+#echo -e "iface {{ OSA2_ENCNO }}.1295 inet static" >> /target/etc/network/interfaces.d/private_int; \
+#echo -e "        address {{ OSA2_ADDRESS }}" >> /target/etc/network/interfaces.d/private_int; \
+#echo -e "        netmask 255.255.255.0" >> /target/etc/network/interfaces.d/private_int; \
+#echo -e "        network 10.20.95.0" >> /target/etc/network/interfaces.d/private_int; \
+#echo -e "        broadcast 10.20.95.255" >> /target/etc/network/interfaces.d/private_int; \
+#echo -e "        vlan-raw-device {{ OSA2_ENCNO }}" >> /target/etc/network/interfaces.d/private_int; \
+#echo -e "        up ip route add 10.20.0.0/16 via 10.20.95.254 || true" >> /target/etc/network/interfaces.d/private_int; \
+#echo -e "" >> /target/etc/network/interfaces.d/private_int; \
+#echo -e "auto {{ OSA2_ENCNO }}.1299" >> /target/etc/network/interfaces.d/private_int; \
+#echo -e "iface {{ OSA2_ENCNO }}.1299 inet manual" >> /target/etc/network/interfaces.d/private_int; \
```

Comment:
And yes, all the hardcoding in this section is trash.. I also think it would be good to be able to specify bridge or no bridge on the given OSA devices, and while we're on calling them out - the primary root disk size is hardcoded as well.